strapi/strapi
# fix(upload): add populate query to findOne method

### What does it do?

It uses the query parameter "populate" to pass to the findOne method.

### Why is it needed?

#9308 
The related field was not populated when getting a file.
I tested on `/upload/files?populate=*&fields=*` and it works, maybe it was fixed by another change.

The `populate` query parameter wasn't handled in the `/upload/files/:file_id`.

### How to test it?

- Create several entities with media attributes
- Create several entities with components with media attributes
- Assign the same file to those entities/components
- Get the file through `/upload/files/{id of the file}?populate=*`
- Check the related attribute

### Related issue(s)/PR(s)

* resolves #9308
